aidies-project/client/src/App.js

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
-import { BrowserRouter, Switch, Route, NavLink } from 'react-router-dom';
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
+import {  useSelector } from 'react-redux';
 
-import UserList from './components/UsersList';
-
-
-function App() {
+import Navigation from './components/Navigation';
+import LoginPanel from './components/LoginPanel';
+import SignupPanel from './components/SignupPanel';
+import CampaignBrowser from './components/CampaignBrowser';
 
+const PrivateRoute = ({component: Component, ...rest}) => (
+  <Route {...rest} render={(props) => (
+    rest.needLogin === true
+      ? <Redirect to='/login' />
+      : <Component {...props} />
+  )} />
+)
+const App = () => {  
+  // const [loaded, setLoaded] = useState(false);
```

Comment:
Most likely a leftover comment from when you were testing something before!

Reply:
Yes, will get rid of it, thanks!